src/store/getters.ts

```diff
@@ -72,6 +72,11 @@ export default {
     return (state.entityType === CorpTypeCd.BC_CORPORATION)
   },
 
+  /** Is True if entity is a Sole Proprietorship or General Partnership. */
+  isFirm (state: StateIF): boolean {
+    return (state.entityType === CorpTypeCd.SOLE_PROP || state.entityType === CorpTypeCd.PARTNERSHIP)
+  },
```

Comment:
I'd prefer to see as a local getter in the component that uses it.

If, in the future, we find ourselves using this multiple times, then we can refactor it.

Also, should this check for ULC, CCC, etc?

Reply:
I think ULC,CCC etc comes under corporations. I added this in the getter because this was getting used in both dashboard and FirmsAddressList